packages/expo-sqlite/ios/SQLiteModule.swift

```diff
@@ -87,10 +104,26 @@ public final class SQLiteModule: Module {
 
     if db == nil {
```

Comment:
Would be great if we don't have to unnecessarily increase indentation level. I believe here you can just do 👇 instead
```swift
if let db {
  return db
}
```